x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/hosts/host_risk_tab.cy.ts

```diff
@@ -23,54 +23,7 @@ import { kqlSearch } from '../../../tasks/security_header';
 import { mockRiskEngineEnabled } from '../../../tasks/entity_analytics';
 
 describe('risk tab', { tags: ['@ess'] }, () => {
-  describe('with legacy risk score', () => {
-    before(() => {
-      cy.task('esArchiverLoad', { archiveName: 'risk_hosts' });
-    });
-
-    beforeEach(() => {
-      login();
-      visitWithTimeRange(hostsUrl('allHosts'));
-      // by some reason after navigate to host risk, page is sometimes is reload or go to all host tab
-      // this fix wait until we fave host in all host table, and then we go to risk tab
-      cy.contains('siem-kibana');
-
-      navigateToHostRiskDetailTab();
-    });
-
-    after(() => {
-      cy.task('esArchiverUnload', { archiveName: 'risk_hosts' });
-    });
-
-    it('renders the table', () => {
-      kqlSearch('host.name: "siem-kibana" {enter}');
-      cy.get(HOST_BY_RISK_TABLE_CELL).eq(4).should('have.text', 'siem-kibana');
-      cy.get(HOST_BY_RISK_TABLE_CELL).eq(5).should('have.text', 'Mar 10, 2021 @ 14:51:05.766');
-      cy.get(HOST_BY_RISK_TABLE_CELL).eq(6).should('have.text', '21.00');
-      cy.get(HOST_BY_RISK_TABLE_CELL).eq(7).should('have.text', 'Low');
-    });
-
-    it('filters the table', () => {
-      openRiskTableFilterAndSelectTheCriticalOption();
-
-      cy.get(HOST_BY_RISK_TABLE_CELL).eq(3).should('not.have.text', 'siem-kibana');
-
-      removeCriticalFilterAndCloseRiskTableFilter();
-    });
-
-    it('should be able to change items count per page', () => {
-      selectFiveItemsPerPageOption();
-
-      cy.get(HOST_BY_RISK_TABLE_HOSTNAME_CELL).should('have.length', 5);
-    });
-
-    it('should not allow page change when page is empty', () => {
-      kqlSearch('host.name: "nonexistent_host" {enter}');
-      cy.get(HOST_BY_RISK_TABLE_NEXT_PAGE_BUTTON).should(`not.exist`);
-    });
-  });
-
-  describe('with new risk score', { tags: ['@serverless'] }, () => {
+  describe('with risk score', { tags: ['@serverless'] }, () => {
```

Comment:
Having 2 nested describes now is not truly needed. It might be better to have only one and move the serverless tag to the above.

Reply:
[remove extra describe](https://github.com/elastic/kibana/pull/201810/commits/e81d87c6dcc205c36e992d87046c7e725486175d)